### PR TITLE
Fix unused args

### PR DESCRIPTION
**Pull request by mcocdawc**
Joint PR of all of us to remove unused args (or explicitly ignore them)

- [x] re-enable rest of ruff checks

related to #57 #58

### CODE REVIEW

File: src/quemb/kbe/pbe.py [hunk 1]
```diff
@@ -162,7 +161,6 @@ def __init__(
         self.hci_pt = hci_pt
 
         if not restart:
-            self.mo_energy = mf.mo_energy
```
Comment (mcocdawc) [inline src/quemb/kbe/pbe.py:165]: that's most likely an error

File: src/quemb/shared/external/cphf_utils.py [hunk 2]
```diff
@@ -141,7 +141,7 @@ def get_full_u(C, moe, eri, no, v, u, thresh=1e8):
     return U
 
 
-def get_full_u_batch(C, moe, eri, no, vs, us, thresh=1e10, timing=False):
+def get_full_u_batch(C, moe, eri, no, vs, us, thresh=1e10):
```
Comment (mcocdawc) [inline src/quemb/shared/external/cphf_utils.py:144]: the way `thresh` is set up makes no sense. (has nothing to do with our review)

**Review by mcocdawc — COMMENTED** [review]
let's discuss in person @mscho527

File: src/quemb/shared/external/cphf_utils.py (hunk 2 above)
Comment (mscho527) [reply src/quemb/shared/external/cphf_utils.py:144]: Investigate later -- #63

**Review by mscho527 — APPROVED** [review]
amazing work team @mcocdawc @lweisburn @aalexiu @bhanscam @ShaunWeatherly

**Pull request merged**